ush/gefs_ocnice_extractvars.sh

```diff
@@ -0,0 +1,73 @@
+#! /usr/bin/env bash
+################################################################################
+#   Script:    
+#
+source "${USHgfs}/preamble.sh"
+
+outdirpre=${1}
+varlist=${2}
+dataformat=${3}
+datares=${4}
+datacompress=${5}
+fhout_ocnice=${6}
+cd "${outdirpre}" || true
+
+nh=${FHMIN_GFS}
+while [[ ${nh} -le ${FHMAX} ]];do
```

Comment:
Use arithmetic operators here and replace with for loop since the interval is constant.

Reply:
Done.